src/resources/js/login.js

```diff
@@ -1,3 +1,8 @@
+if (localStorage.auth_failed) {
+    document.getElementById('auth-failed').style.display = 'block';
+    delete localStorage.auth_failed;
```

Comment:
It seems like it's not the standard-compliant way of deleting things. Better use [`removeItem()`](https://developer.mozilla.org/en/docs/Web/API/Storage/removeItem).